app/models/service_ansible_playbook.rb

```diff
@@ -73,4 +90,29 @@ def extra_vars_from_dialog
 
     params.blank? ? {} : {:extra_vars => params}
   end
+
+  def create_inventory(action, hosts)
+    manager(action).with_provider_connection do |connection|
+      inventory = connection.api.inventories.create!(:name => inventory_name(action), :organization => 1)
```

Comment:
You could do:
```ruby
def create_inventory(action, hosts)
  manager(action).with_provider_connection do |connection|
    connection.api.inventories.create!(:name => inventory_name(action), :organization => 1).tap do |inventory|
      hosts.split(',').each do |host|
        connection.api.hosts.create(:name => host, :inventory => inventory.id)
      end
    end
  end
end
```
The `tap` further indents the host creation and better represents the flow.